### ISSUE TITLE
[docs] Update description of Elastic Agent in the integrations documentation

### ISSUE THREAD

**Issue by dedemorton** [elastic/integrations#3690]
We have expanded the description of Elastic Agent to indicate it can collect and forward data from remote services and hardware. The integrations docs need to be updated to reflect this change:

We need to update https://docs.elastic.co/integrations plus the info box that appears on every page:

![image](https://user-images.githubusercontent.com/14206422/178573053-d1014e9a-2a4c-4aa3-ba46-24520142df6a.png)

### Long version:

Elastic Agent is a single, unified way to add monitoring for logs, metrics, and other types of data to a host. It can also protect hosts from security threats, query data from operating systems, forward data from remote services or hardware, and more. A single agent makes it easier and faster to deploy monitoring across your infrastructure. Each agent has a single policy you can update to add integrations for new data sources, security protections, and more.

### Short version:

Elastic Agent is a single, unified way to add monitoring for logs, metrics, and other types of data to a host. It can also protect hosts from security threats, query data from operating systems, forward data from remote services or hardware, and more.

Related to https://github.com/elastic/observability-docs/pull/1977
